counterfactuals.qmd

```diff
@@ -0,0 +1,276 @@
+# Projections and counterfactuals
+
+```{r}
+#| include: false
+#| cache: false
+set.seed(1)
+source("common.R")
+options(monty.progress = FALSE)
+```
+
+## Introduction
+
+```{r}
+library(odin2)
+library(dust2)
+library(monty)
+```
+
+We have seen how to fit `odin` models with `monty` to produce estimates of fitted parameters and the trajectories of the underlying model. It is common to use such outputs for either of the following:
+
+1. **Projections** - running the model forward from the end of fitted time series to estimate what would happen under given choices of parameters.
+2. **Counterfactuals** - estimating what would have happened had parameters been different from those estimated from or assumed in the fits. This could be run from the initial time used in fitting, or from a point in the data time series.
+
```

Comment:
Perhaps add a sentence here foreshadowing the approach? Something along the lines of:

"Both of these require that you configure your likelihood (i.e., the object created with `dust_likelihood_monty`) to save additional information about the filtering process."